### PR TITLE
Update upbeat.py

### PR DESCRIPTION
**Pull request by mystxii**
hi k bye

### CODE REVIEW

File: src/upbeat.py [hunk 1]
```diff
@@ -39,6 +39,13 @@ def spotifyid():
             else:
                 f = t
             return f 
+        def deezerId():
```
Comment (DylsterDev) [inline src/upbeat.py:42]: Has this been tested?

Comment (DylsterDev) [inline src/upbeat.py]: Have you tested this endpoint at all?

**Review by DanielDot — APPROVED** [review]
Everything now looks good.

**Comment by DylsterDev** [comment]
Yeah this looks fine

**Review by DylsterDev — APPROVED** [review]
I'm happy with this being merged. It looks like it has minimal errors

**Pull request merged**